app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+class CommentsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :set_comment, only: %i[show edit update destroy]
+
+  # GET /comments
+  def index
```

Comment:
Just one thought, do we want an index view for comments, and if so, do we want to restrict it to admins?